server/src/services/apps.service.ts

```diff
@@ -290,6 +290,14 @@ export class AppsService {
   }
 
   async createVersion(user: User, app: App, versionName: string, versionFromId: string): Promise<AppVersion> {
+    if (!versionName) {
```

Comment:
Do we need the validations here? since we have it in the [dto](https://github.com/ToolJet/ToolJet/pull/3014/files#diff-e6f45f54d8d800f55022f14e4be153a11cb7088acf919080a4439012c1517ddbR8)

Reply:
Currently `createVersion` is invoked from `app controller` [Line](https://github.com/ToolJet/ToolJet/blob/develop/server/src/controllers/apps.controller.ts#L261) and `VersionCreateDto` is added to this controller but what if some other `service/controller` uses this `createVersion` function without proper `check/DTO` in those might cause some issue